### DISCUSSION TITLE
[Improvement] Internationalization

### DISCUSSION THREAD

**Discussion by cornucopib** [Kong/insomnia#3724] [Ideas]
- Insomnia Version: ___
- Operating System: ___

**Comment by welcome** [comment]
👋 Thanks for opening your first issue! If you're reporting a 🐞 bug, please make sure
you include steps to reproduce it. If you're requesting a feature 🎁, please provide real
use cases that would benefit. 👪

To help make this a smooth process, please be sure you have first read the
[contributing guidelines](https://github.com/getinsomnia/insomnia/blob/master/CONTRIBUTING.md).

**Comment by gschier** [comment]
I think internationalization is a great idea as Insomnia gets more popular.

Can you elaborate on what parts of the app should be covered first?

**Comment by wdawson** [comment]
Hi everyone! We started a poll here #3497 to help us figure out what's most important to all of you. As a result, I'll be locking this conversation. Thanks, @Vector-SMG for suggesting 2 years ago!
